cpp/locality/Voronoi.cc

```diff
@@ -1,195 +1,170 @@
-
-                // Bonds where a ridge goes to infinity (index-1) have weight 0
-                bool weighted_bond = true;
-                std::vector< vec3<double> > current_ridge_vertex;
-                for (int ridge_vert_id = ridge_vertex_indices[ridge]; ridge_vert_id < ridge_vertex_indices[ridge+1]; ++ridge_vert_id)
-                {
-                    if (ridge_vertices[ridge_vert_id] == -1)
-                    {
-                        add_valid_bonds(local_bonds, i, expanded_ids[i], j, expanded_ids[j], N, 0, distance);
-                        weighted_bond = false;
-                        break;
-                    }
-                    else
-                    {
-                        current_ridge_vertex.push_back(vertices[ridge_vertices[ridge_vert_id]]);
-                    }
-                }
-
-                if (weighted_bond)
-                {
-                    if (box.is2D())
-                    {
-                        // 2D weight is the length of the ridge edge
-                        vec3<double> v1 = current_ridge_vertex[0];
-                        vec3<double> v2 = current_ridge_vertex[1];
-                        // not necessary to have double precision in weight calculation
-                        vec3<float> r_ij(box.wrap(v1 - v2));
-                        weight = sqrtf(dot(r_ij, r_ij));
-                    }
-                    else
-                    {
-                        // 3D weight is the area of the ridge facet
-                        // Create a vector of all vertices for this facet
-                        std::vector< vec3<double> > vertex_coords;
-
-                        for (std::vector< vec3<double> >::iterator ridge_vert_id = current_ridge_vertex.begin();
-                            ridge_vert_id != current_ridge_vertex.end();
-                            ++ridge_vert_id)
-                        {
-                            vec3<double> vert = *ridge_vert_id;
-                            vertex_coords.push_back(vert);
+        m_polytopes.resize(n_points);
+        m_volumes.prepare(n_points);
+
+        vec3<float> boxLatticeVectors[3];
+        boxLatticeVectors[0] = box.getLatticeVector(0);
+        boxLatticeVectors[1] = box.getLatticeVector(1);
+        if (box.is2D()) {
+            boxLatticeVectors[2] = vec3<float>(0, 0, 1);
+        } else {
+            boxLatticeVectors[2] = box.getLatticeVector(2);
+        }
+        voro::container_periodic container(
+            boxLatticeVectors[0].x,
+            boxLatticeVectors[1].x,
+            boxLatticeVectors[1].y,
+            boxLatticeVectors[2].x,
+            boxLatticeVectors[2].y,
+            boxLatticeVectors[2].z,
+            3, 3, 3, 3
+        );
+
+        for (size_t query_point_id = 0; query_point_id < n_points; query_point_id++) {
+            container.put(query_point_id, points[query_point_id].x, points[query_point_id].y, points[query_point_id].z);
+        }
```

Comment:
I'm personally inclined to avoid using the `query_point` terminology at all in this code, since Voronoi constructions don't really make sense for two distinct sets of points. I could be convinced otherwise, though, if you think it's a helpful mental model. Thoughts?

Reply:
I also don't think it's a good enough reason to hold up this PR. So if nothing else major crops up I'm not going to bother asking for a change no matter where we come down.

Reply:
I'm just trying to keep it consistent. I don't know what else I would call it without being _more_ strange, since we've standardized terms elsewhere in the codebase.